src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,41 @@
+.calendar {
```

Comment:
Add some space for your calendar to avoid this behavior (day is cut on hover)
![Screenshot 2023-02-28 at 18 38 45](https://user-images.githubusercontent.com/117587463/221919546-e76ade3f-2582-4580-a012-92ebd58de92a.png)